visualizations/podium/components/PodiumPlace.tsx

```diff
@@ -1,77 +1,145 @@
 import React from "react";
-import { StackItem } from 'nr1'
-import PlacingIndicator from './PlacingIndicator';
+import { StackItem } from "nr1";
+import PlacingIndicator from "./PlacingIndicator";
+import { ImageWithFallback } from "../../../shared/ImageWithFallBack";
 
 type AttributesListProps = {
-    placing: any;
-    caption: any;
-    subCaption: any;
-    link: any;
-    circles: any;
-    value: any;
-    placingOffset: any;
-    placingColor: any;
-    decorationColor: any;
-    captionColor: any;
-    subCaptionColor: any;
-    valueColor: any;
-    imageUrl: any;
+  placing: any;
+  caption: any;
+  subCaption: any;
+  link: any;
+  circles: any;
+  value: any;
+  placingOffset: any;
+  placingColor: any;
+  decorationColor: any;
+  captionColor: any;
+  subCaptionColor: any;
+  valueColor: any;
+  imageUrl: any;
 };
 
-const PodiumPlace =  ({ placing, value, caption, subCaption, placingOffset, link, placingColor, decorationColor, captionColor, subCaptionColor, valueColor, imageUrl, circles }: AttributesListProps) => {
-  const placingOffsetWidth=0.8;
+const PodiumPlace = ({
+  placing,
+  value,
+  caption,
+  subCaption,
+  placingOffset,
+  link,
+  placingColor,
+  decorationColor,
+  captionColor,
+  subCaptionColor,
+  valueColor,
+  imageUrl,
+  circles,
+}: AttributesListProps) => {
+  const placingOffsetWidth = 0.8;
 
   let imageThumbnail, imageThumbnailCircle;
-  if(imageUrl && imageUrl!="")  {
-    imageThumbnail=<div className="imageThumb"><img src={imageUrl} alt="name"/></div>
-    imageThumbnailCircle=<div className="imageThumb"><img className="circle" style={{borderColor: decorationColor}} src={imageUrl} alt="name"/></div>
+  if (imageUrl && imageUrl != "") {
+    imageThumbnail = (
+      <div className="imageThumb">
+        <ImageWithFallback src={imageUrl} alt={name} />
```

Comment:
`name` prop is not being passed down or provided by a hook

Reply:
done